glide.lock

```diff
@@ -1,5 +1,5 @@
 hash: 72fc48b4e2b9e8116ca8457e10d1eb0df1c1d1e7d0286f75928f0eb427a9e46c
-updated: 2017-05-02T18:53:14.187450044+08:00
+updated: 2017-05-09T16:49:34.415088297+08:00
```

Comment:
we will need json functions defiend in tipb, so we must `make update` soon or later.
so I update this file here in order to reduce next PR size. :)